### DISCUSSION TITLE
Is it possible to not load certain services?

### DISCUSSION THREAD

**Discussion by Stainless2k** [TypeFox/monaco-languageclient#625] [Q&A] [answered]
When using `<MonacoEditorReactComp />`  in extended mode, a lot of services get loaded, but I don't need many of them.

e.g I don't need most file related services and having them even throws some errors in certain situations.

**Comment by kaisalmen** [comment]
Hi @Stainless2k the default services are specified [here](https://github.com/TypeFox/monaco-languageclient/tree/main/packages/wrapper#configuration). layout, environment, extension, files and quickAccess are alwas loaded by `@codingame/monaco-vscode-api`. You always require those. `monaco-languageclient` adds model and langauges, `monaco-editor-wrapper` adds configuration, plus theme and textmate in extended mode.

**Reply by Stainless2k** [reply]
So it's not possible to not load/disable certain services? I assume to fix my issues with the file service I would have to use "@codingame/monaco-vscode-files-service-override" ?

**Reply by kaisalmen** [reply, marked as answer]
It does it internally. Some services rely on each other and are required.
